tests/pytorch/nightly/dlrm.libsonnet

```diff
@@ -19,45 +19,44 @@ local tpus = import 'templates/tpus.libsonnet';
 local utils = import 'templates/utils.libsonnet';
 
 {
-  local command_common = |||
-    pip install onnx
-    git clone --recursive https://github.com/pytorch-tpu/examples.git
-    python examples/deps/dlrm/dlrm_tpu_runner.py \
-      --arch-sparse-feature-size=64 \
-      --arch-mlp-bot=512-512-64 \
-      --arch-mlp-top=1024-1024-1024-1 \
-      --arch-interaction-op=dot \
-      --lr-num-warmup-steps 10 \
-      --lr-decay-start-step 10 \
-      --num-batches=1000 \
-      --data-generation="random" \
-      --numpy-rand-seed=727 \
-      --print-freq 100 \
-      --num-indices-per-lookup=100 \
-      --use-tpu \
-      --metrics-debug \
-      --num-indices-per-lookup-fixed \
-  |||,
   local dlrm = common.PyTorchTest {
+    local config = self,
     modelName: 'dlrm',
 
     volumeMap+: {
       datasets: common.datasetsVolume,
     },
-    podTemplate+:: {
-      spec+: {
-        containerMap+: {
-          train+: {
-            resources+: {
-              requests: {
-                cpu: '9.0',
-                memory: '30Gi',
-              },
-            },
-          },
-        },
-      },
+    paramsOverride:: {
+      scriptPath: 'tpu-examples/deps/dlrm/dlrm_tpu_runner.py',
+      miniBatchSize: 256,
+      archEmbeddingSize: '1000000-1000000',
+      tpuModelParallelGroupLen: 1,
+      tpuCores: 1,
```

Comment:
This should default to `config.accelerator.numCores` so we get the correct number of cores for TPU v4: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/224065875e2505ddbd26bcfcc2c6ccdddb8d3ab8/templates/tpus.libsonnet#L25

Remove `tpuCores: 8` from `seq_fwd`, `mp_fwd`, etc too, since that will only work on v3